### PR TITLE
[Site] Updated docs section style

### LINKED ISSUE #124: [Site] Change color and style of menu items
**Current State of the GetNighthawk.dev site**
The current implementation of the following sections should be corrected to follow the design specification:
1. the FAQs section of [getnighthawk.dev](https://getnighthawk.dev/)

FAQs

2. The sidebar of [Getnighthawk Docs](https://getnighthawk.dev/docs)

Docs SideNav

**Desired State of the GetNighthawk.dev site**

The design should be similar to this for the FAQs section: 
FAQs

Similar design should also be implemented for the Docs side-navbar.

For convenience, you can check out the [Figma design](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

---

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by NikhilSharma03**
Signed-off-by: Nikhil <nikhilsharmamusic2000@gmail.com>

**Description**
Updated style and color of docs section.

## After Changes
Screenshot from 2021-06-03 17-55-52

This PR fixes #124 

**Notes for Reviewers**
Let me know if there are any other changes or fixes we can make.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by NikhilSharma03** [comment]
@vinayaksh42 @JubayerJoy Can you please review the changes? Thank you.

**Review by JubayerJoy — COMMENTED** [review]
@NikhilSharma03 really nice work! 🚀
Some suggestions,
1. The menu is not closing on click, please change it so it collapses on click.
2.  Keeping the first menu item open by default would look nicer and give good UX.

**Comment by Anita-ihuman** [comment]
> 1. The menu is not closing on click, please change it so it collapses on click.

well, I think keeping the dropdown closed is better tho. This is because by the time the details, expected to be on the doc page are complete, the page would look too busy having the dropdown open by default.

**Review by Anita-ihuman — COMMENTED** [review]
Thank you for taking this up.
 it would be great if the dropdown closes back on clicking the button.
right now it just Opens.

**Comment by leecalcote** [comment]
@NikhilSharma03, are you trying to tell the user that "there's no going back"? ;)

**Comment by NikhilSharma03** [comment]
@Anita-ihuman @JubayerJoy 
Changes to make :
- Dropdown closes back on clicking the button
- Keeping the dropdown closed
Can you please confirm the changes so I can work on them. Thank you.

**Comment by Anita-ihuman** [comment]
> @Anita-ihuman @JubayerJoy
> Changes to make :
> 
> * Dropdown closes back on clicking the button
> * Keeping the dropdown closed
> 
> Can you please confirm the changes? Thank you.

Yes, Those are the changes to be made.

**Comment by NikhilSharma03** [comment]
@Anita-ihuman @JubayerJoy Can you please review the changes.

File: docs/_sass/getnighthawk.scss [hunk 1]
```diff
@@ -5977,7 +5977,51 @@ div.globalFooterCard.card-environment {
   color: #696969;
 }
 
-.sidebar {
+.sidebar  {
```
Comment (vinayaksh42) [inline docs/_sass/getnighthawk.scss:5980]: Removal of this empty class.

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]

File: docs/_sass/getnighthawk.scss (hunk 1 above)
Comment (NikhilSharma03) [reply docs/_sass/getnighthawk.scss:5980]: Yeah Sure.

**Review by vinayaksh42 — APPROVED** [review]
Great work @NikhilSharma03

**Review by Anita-ihuman — APPROVED** [review]
Looks good to me

**Review by JubayerJoy — APPROVED** [review]
Really Nice work @NikhilSharma03 🚀
LGTM 🎈

**Pull request merged**